op_tests/op_benchmarks/triton/l2cache.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
```

Comment:
This is useful. But it can probably be renamed to have mha in its name. I don't know this dir is the right place for this and other test files. What's @rahulbatra85 's opinion on this?